custom/icds_reports/ucr/data_sources/child_cases_monthly_v2.json

```diff
@@ -472,6 +508,10 @@
               "type": "named",
               "name": "female"
             },
+            {
```

Comment:
This question is more global, but`is_resident` was added as a filter to a large number of indicators here. That filter isnt really used in the dashboard at all (you can filter by it if you choose for some things but its not applied by default). Are we concerned about differences in the data that this will create?

Reply:
Sunaina/Akshita can you respond to this?